scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedJvmTaskExecutionCommandCreator.java

```diff
@@ -177,6 +180,12 @@
         return javaCommand;
     }
 
+    private PAPropertyString createNodeSourceProperty() {
+        PAPropertyString nodeSourceNameProperty = new PAPropertyString("proactive.node.nodesource", false, "local");
```

Comment:
the default value for a node source not specified is "Default" (when the node is deployed using the proactive-node command or through an agent without speicifying the node-source parameter)

Reply:
Didn't know that. I will fix it. Thanks.